serverless/serverless-kafka.adoc

```diff
@@ -12,6 +12,31 @@ include::modules/technology-preview.adoc[leveloffset=+2]
 You can use the `KafkaChannel` channel type and `KafkaSource` event source with {ServerlessProductName}.
 To do this, you must install the Knative Kafka components, and configure the integration between {ServerlessProductName} and a supported link:https://access.redhat.com/documentation/en-us/red_hat_amq/7.6/html/amq_streams_on_openshift_overview/index[Red Hat AMQ Streams] cluster.
 
+The {ServerlessOperatorName} provides the Knative Kafka API that can be used to create a `KnativeKafka` custom resource:
+
+.Example `KnativeKafka` custom resource
+[source,yaml]
+----
+apiVersion: operator.serverless.openshift.io/v1alpha1
+kind: KnativeKafka
+metadata:
+    name: knative-kafka
+    namespace: knative-eventing
+spec:
+    channel:
+        enabled: true <1>
+        bootstrapServers: <bootstrap_server> <2>
+    source:
+        enabled: true <3>
+----
+<1> Enables developers to use the `KafkaChannel` channel type in the cluster.
+<2> A comma-separated list of bootstrapped servers from your AMQ Streams cluster.
+<3> Enables developers to use the `KafkaSource` event source type in the cluster.
+
 // Install Kafka
 include::modules/serverless-install-kafka-odc.adoc[leveloffset=+1]
-include::modules/serverless-install-kafka-yaml.adoc[leveloffset=+1]
+
+[id="serverless-kafka-next-steps"]
```

Comment:
Same for this one, does this need `_{context}` or is this not required for titles in assembly files?

Reply:
Just for modules afaik